hack/make-rules/verify.sh

```diff
@@ -21,15 +21,10 @@ set -o pipefail
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
 source "${KUBE_ROOT}/cluster/lib/util.sh"
 
-if [ -n "${VERBOSE}" ]; then
```

Comment:
The default behavior is fine, but it may be best to remove it from these scripts to avoid confusion in the future (as it is no longer a valid option for "verify", if this PR is merged)